plugins/postgres/dbt/include/postgres/macros/adapters.sql

```diff
@@ -91,3 +91,19 @@
 {% macro postgres__current_timestamp() -%}
   now()
 {%- endmacro %}
+
+{% macro postgres__archive_get_time() -%}
+  {{ current_timestamp() }}::timestamp without time zone
+{%- endmacro %}
+
+{% macro postgres__make_temp_relation(base_relation, suffix) %}
```

Comment:
@beckjake check this out - I override the `make_temp_relation` macro for pg/redshift to just return a Relation with an identifier. This means that we can use this temp relation to:
1. Build a temp table using `create_table_as(True, tmp_relation)`
2. Query the relation directly
3. Pass it around to adapter methods